packages/experiments/src/components/Tile/Tile.tsx

```diff
@@ -62,16 +62,19 @@ export const TileLayoutSizes: {
  * @class Tile
  * @extends {React.Component<ITileProps, ITileState>}
  */
-export class Tile extends BaseComponent<ITileProps, ITileState> {
+export class Tile extends React.Component<ITileProps, ITileState> {
   private _nameId: string;
   private _activityId: string;
   private _labelId: string;
   private _descriptionId: string;
+  private _events: EventGroup;
 
   // tslint:disable-next-line:no-any
   constructor(props: ITileProps, context: any) {
     super(props, context);
 
+    initializeComponentRef(this);
```

Comment:
Possibly also initializeFocusRects? (same in the other components, at least the top-level ones)

Reply:
Seems reasonable to mitigate any potential for regression. I will post an iteration as well as fix the failing tests in a bit.

Thank you for catching the missing disposals. Forgot only `initializeComponentRef` cleans-up after itself.

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/pull/9810/commits/656f5927f642ba7ab44302c4e2d93ddc37eb79e7 adds invocations of `initializeFocusRects` in top-level components that have inheritance of `BaseComponent` removed including removals from #9765.

My rationale for not including `ShimmeredDetailsList`:

1. It does not include focus-able content
1. Wraps a `DetailsList` that should invoke `initializeFocusRects` once `BaseComponent` inheritance is removed.